src/SupervisionSuccess/constants.js

```diff
@@ -1,2 +1,4 @@
-/* eslint-disable import/prefer-default-export */
-export const CSV_PROCESSING_ERROR = "Error occurred while processing csv file";
+export const ERROR_RESPONSE_NOT_OK = "Could not load data file (response not OK)";
```

Comment:
Let's change this to: "An error happened while attempting to load the params file. Response was: [whatever]" and then update the code that uses this error message to insert the actual response code that was not okay into those square brackets.